python/python3-enzyme: Version bump to 0.5.1

bassmadrigal opened the pull request.

bassmadrigal commented:
@sbo-bot: build python/python3-enzyme

willysr commented:
@bassmadrigal this failed to build

willysr commented on python/python3-enzyme/python3-enzyme.info:
```diff
@@ -1,8 +1,8 @@
+MD5SUM="a3a9abfc7e590b10221b800996b90644"
 DOWNLOAD_x86_64=""
 MD5SUM_x86_64=""
 REQUIRES=""
```
you need python3-wheel and python3-build for this

bassmadrigal replied:
Oops, I caught this after I committed the change and reverted the commit locally and created a new one, but I forgot to push it to GitHub.

willysr reviewed (commented):
please update the deps

bassmadrigal commented:
@sbo-bot: build python3-enzyme

RSKYS commented:
@bassmadrigal Discord update again xD
Also could you please try building Picard on next release to see if you encounter issues?

Cheers

bassmadrigal commented:
> @bassmadrigal Discord update again xD 

I don't typically need reminders for discord and already submitted an update and had it merged (#6898). It'll be in the next public update this weekend. I almost always verify the latest discord is submitted before the SBo public update is released.

If you don't want to run into problems when discord is updated until the public update, please follow the instructions on the README for discord on SBo to have it ignore the version when starting.

>Also could you please try building Picard on next release to see if you possibly encounter any issues?

I'm not sure what you mean. Picard is already at it's latest version (2.11) and was tested when I submitted it back in February. If you are seeing issues running it, please send me an email about it rather than make comments in unrelated PRs.

bassmadrigal commented:
@sbo-bot: build python/python3-enzyme

RSKYS commented:
> I don't typically need reminders for discord and already submitted an update and had it merged (https://github.com/SlackBuildsOrg/slackbuilds/pull/6898). It'll be in the next public update this weekend. I almost always verify the latest discord is submitted before the SBo public update is released.
> 
> If you don't want to run into problems when discord is updated until the public update, please follow the instructions on the README for discord on SBo to have it ignore the version when starting.

Ah I see, thanks!
Once again I'm sorry for what happened before, I'm trying to be nice; honesty don't know what to do with people who experienced bad behavior from me before.

> I'm not sure what you mean. Picard is already at it's latest version (2.11) and was tested when I submitted it back in February. If you are seeing issues running it, please send me an email about it rather than make comments in unrelated PRs.

Oh yes your update is not an issue, I've updated python3-fasteners.
It should be fine but just in case something goes wrong I said; highly unlikely.

bassmadrigal commented:
> > I don't typically need reminders for discord and already submitted an update and had it merged (#6898). It'll be in the next public update this weekend. I almost always verify the latest discord is submitted before the SBo public update is released.
> > If you don't want to run into problems when discord is updated until the public update, please follow the instructions on the README for discord on SBo to have it ignore the version when starting.
> 
> Ah I see, thanks! Once again I'm sorry for what happened before, I'm trying to be nice; honesty don't know what to do with people who experienced bad behavior from me before.

My response had nothing to do with your previous behavior. Discord is just updated frequently these days and I always make sure to stay on top of it so I don't get inundated with requests to update it. If it goes more than a week after a new release without an update, feel free to email me to check on it.

It's also why I updated the README to make it so people can continue to run it until the next public update is pushed.

> > I'm not sure what you mean. Picard is already at it's latest version (2.11) and was tested when I submitted it back in February. If you are seeing issues running it, please send me an email about it rather than make comments in unrelated PRs.
> 
> Oh yes your update is not an issue, I've updated python3-fasteners. It should be fine but just in case something goes wrong I said; highly unlikely.

Generally, it should be on the maintainer pushing the update to make sure the packages that depend on it still work. You can use the Reverse dependencies part of the advanced search on SBo to check what programs rely on yours. If there are a lot of packages that rely on it, some will notify maintainers of the impending update and ask them to check. 

However, in my experience, python packages are usually pretty flexible and version updates don't typically cause issues. Usually release notes for those programs will state if there are any potential breakages with the update to give you a heads up to check.

willysr commented:
merged
